### PR TITLE
Complex Cycling Dashboard

### PR DESCRIPTION
**Pull request by kvr000**
This includes:
- Customizable dashboards (though still hardcoded in resources for now)
- Map gauge
- DateTime gauge
- Temperature gauge
- Grade with elevation gauge
- Lon Lat gauge
- Saving and loading chosen dashboard code to/from project files.

Video sample: https://youtu.be/0Xs88mh2QAc

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -124,8 +125,11 @@ libraryDependencies += "com.jgoodies" % "looks" % "2.2.2"
 libraryDependencies += "com.jgoodies" % "jgoodies-common" % "1.8.1"
 libraryDependencies += "com.miglayout" % "miglayout" % "3.7.4"
 libraryDependencies += "xuggle" % "xuggle-xuggler" % "5.4" from "https://files.liferay.com/mirrors/xuggle.googlecode.com/svn/trunk/repo/share/java/xuggle/xuggle-xuggler/5.4/xuggle-xuggler-5.4.jar"
+libraryDependencies += "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % jacksonVersion
+libraryDependencies += "com.fasterxml.jackson.module" %% "jackson-module-scala" % jacksonVersion
 libraryDependencies += "org.json4s" %% "json4s-native" % json4sVersion
 libraryDependencies += "org.json4s" %% "json4s-jackson" % json4sVersion
+libraryDependencies += "org.json4s" %% "json4s-jackson" % json4sVersion
```
Comment (peregin) [inline build.sbt:132]: it duplicates the previous line, it can be removed

File: src/main/scala/peregin/gpv/format/GpsFormatter.scala [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package peregin.gpv.format
+
+
+object GpsFormatter {
+
+  def formatLon(lon: Double): String = {
+    return formatAxis("WE", lon)
```
Comment (peregin) [inline src/main/scala/peregin/gpv/format/GpsFormatter.scala:7]: minor nitpicking, in `Scala` is not necessary to add the `return` keyword, the return value is always the evaluation of the last expression or value (is used explicitly for early returns).
e.g. this is usually expressed as `def formatLon(lon: Double): String = formatAxis("WE", lon)`

**Review by peregin — APPROVED** [review]
great job @kvr000 👏 
Thank you for the impressive amount of changes.
I hope you liked working with `Scala`.
Great video editing, checked it on YouTube, awesome 🚀 
Happy cycling!

File: build.sbt (hunk 1 above)
Comment (kvr000) [reply build.sbt:132]: Oh, thanks, fixed in next PR.

**Comment by kvr000** [comment]
Glad to help, @peregin .  Thanks for this app!

**Pull request merged**